app/components/documents/linked-document-link.hbs

```diff
@@ -87,7 +143,7 @@
                         <EmberTooltip @side="bottom" @tooltipClass="auk-tooltip">
                           {{t "document-not-consultable"}}
                         </EmberTooltip>
-                        {{piece.name}}{{#if piece.file}}.{{piece.file.extension}}{{/if}}
+                        {{piece.name}}
```

Comment:
explanation for this: this is the else part of `{{#if piece.file}}` so file never exists here.